### ISSUE TITLE
Layout issue: arrows in latest posts boxes

### ISSUE THREAD

**Issue by tobiasfabian** [smashingmagazine/redesign#254]
**- Do you want to request a *feature* or report a *bug*?**
bug

**- What is the current behavior?**
The ”1.044 smashing members“ box (`.col--article--meow__membership`) is very high. That’s why the link arrows in the other boxes (`.col--article--grid`) are far away from the corresponding text. It’s not super obvious where the arrows belong to.

![screenshot-bug](https://user-images.githubusercontent.com/1524319/33118878-8490af6a-cf6d-11e7-9cef-fdac133f9654.png)

**- If the current behavior is a bug, please provide the steps to reproduce.**
Open home page on a large screen (> 1460px)

**- What is the expected behavior?**
Arrow should be directly beneath text.
A fix is simple. Just add `align-item: flex-start;` to `.article--grid__container`.

![screenshot-fixed](https://user-images.githubusercontent.com/1524319/33118857-70cbfe30-cf6d-11e7-9b14-6c7c18005075.png)

**- Please mention your operating system version and the version of your browser**
Safari (Browser width ~1600), macOS. Same behavior in Chrome and Firefox (tested on macOS).

**Comment by vitalyfriedman** [comment]
Good point @tobiasfabian — let me see how we'll adjust the layout. Good idea!
